test/dependencies/Makefile

```diff
@@ -0,0 +1,26 @@
+.PHONY: test
+
+REACT_VERSION ?= 18
+SLATE_VERSION := $(shell cat ../../package.json | grep '"slate":' | cut -d'"'  -f4)
+SLATE_REACT_VERSION := $(shell cat ../../package.json | grep '"slate-react":' | cut -d'"'  -f4)
+SLATE_HISTORY_VERSION := $(shell cat ../../package.json | grep '"slate-history":' | cut -d'"'  -f4)
+
+test: clean app/package.json
```

Comment:
Should we explain it in the README?

Reply:
The `make` commands if someone wants to run locally, but I realized that's not needed.  That would be too much unnecessary information :)